base_tier_validation/tests/test_tier_validation.py

```diff
@@ -941,7 +938,7 @@ def test_27_reevaluate_validation(self):
         reviews = test_record.with_user(self.test_user_2.id).reevaluate_reviews()
         # Check need validation
         self.assertTrue(test_record.need_validation)
-        self.assertEqual(len(reviews), 2)
+        self.assertEqual(len(reviews), 1)
```

Comment:
Are these changes related to not calling `_update_counter`. If so, the name is misleading...

Reply:
No, all these changes are related to using `_invalidate_cache`, it is something I don't like, but it is related to the `review_ids` field being a One2many and the inverse_name to which it refers (`res_id`) being an integer field and not a many2one, that would simplify things (we have already discussed that casuistry in some occasion), but I don't want to open that debate now.

Reply:
Uhm, isn't `Many2oneReference` field type for that?

Reply:
Ok, thanks, changed. The test changes are still needed, although I have added some validations that help to understand it better.